resolwe_bio/processes/modeling/pathway_predictor.yml

```diff
@@ -1,13 +1,16 @@
-- slug: pathway-predictor
+- slug: pathways-predictor
   name: Predict heterologous metabolic pathways
   requirements:
     expression-engine: jinja
     executor:
       docker:
         image: dddecaf/resolwe-pathway-predictor:latest
```

Comment:
`dddecaf/resolwe-pathway-predictor`

latest is the default, thus not needed here